priv/apns/dev_cert.pem

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
They can be generated but it adds one and only one additional step to building the project. If I leave those example certs in repo, there not need to have openssl while running the project and there not need for Makefile.